apps/docs/src/app/platform/component-docs/platform-button/platform-button-docs.component.ts

```diff
@@ -27,12 +27,19 @@ export class PlatformButtonDocsComponent implements OnInit {
                         type: 'string',
                         enum: ['', 'standard', 'positive', 'transparent', 'negative', 'emphasized', 'ghost']
                     },
-                    compact: {
-                        type: 'boolean'
+                    contentDensity: {
```

Comment:
let's align with core

Reply:
Hi @fkolar , base.ts has content Density for platform components. Please let me know your suggestion on this. So that I can update other comments accordingly.

Reply:
Even I have updated the title to go back to WIP mode so we can sync on the docs. we choose the name very carefully. 

This had its own evolution, where we started with size which conflicted with native attributes and then we try to find real meaning what compact and cozy really is. Developer can pass clear values `compact / cozy`.  If there will be in the future 3 option, like fluid , then have a type is safe approach. 

In the platform there are several differences on namings trying to put structure across all component.  

If we are able to find better name we are open to this. the` compact` just does not sound right for platform level. It needs to be a type with 2 possible values.

Reply:
With respect to platform component consistency will be having it as content density.

Reply:
With respect to platform component consistency will be having it as content density.

Reply:
How about we align with the core where we decided to use input `compact`?

Reply:
I need confirmation from Frank to this change.

Reply:
We try to call the things right name even `contentDensity` might not be the best one, and I would like something with size we are just not comportable with just boolean compact. 

This way we could go to other names, where in combobox there is property dropdownValues which to me a its bit confusing and we call this differently as well. And there are many differences. 

We are definitelly open to suggestions maybe sizeFactor, but density is comon term. E.g. material has 3 values: 
https://material.io/design/layout/applying-density.html#components